public/app/features/dashboard/components/PanelEditor/PanelEditorTabs.tsx

```diff
@@ -41,7 +41,7 @@ export const PanelEditorTabs: FC<PanelEditorTabsProps> = React.memo(({ panel, da
       <TabsBar className={styles.tabBar} hideBorder>
         {tabs.map((tab) => {
           if (tab.id === PanelEditorTabId.Alert) {
-            renderAlertTab(tab, panel, dashboard, onChangeTab);
+            return renderAlertTab(tab, panel, dashboard, onChangeTab);
```

Comment:
Sneaky bug

Reply:
There is an interesting Eslint rule which could help in such cases https://eslint.org/docs/rules/consistent-return

Reply:
That sounds interesting; though I imagine it will need a lot of investment to add that rule so I'm not too keen on it :)

Reply:
Yep, I think the same :)